input/docs/handbook/when-activated/index.md

```diff
@@ -51,7 +51,7 @@ public class ActivatableViewModel : IActivatableViewModel
 
 ## How to ensure a view model gets activated?
 
-The framework will acknowledge the link from your `ViewModel` to your `View` if the latter implements the [IViewFor interface](../../getting-started/#create-views). Remember to make your ViewModel a `DependencyProperty` and to add a call to `WhenActivated` in your `IViewFor<TViewModel>` implementation constructor.
+The framework will acknowledge the link from your `ViewModel` to your `View` if the latter implements the [IViewFor interface](https://www.reactiveui.net/docs/handbook/view-location/extending-iviewfor). Remember to make your ViewModel a `DependencyProperty` and to add a call to `WhenActivated` in your `IViewFor<TViewModel>` implementation constructor.
```

Comment:
If you can change this not to be a full URL that would be great thanks. 

eg 

`../../handbook/view-location/extending-iviewfor`

Thanks,

Reply:
Oh sure -  totally overlooked that one 🙈

Reply:
Fixed